.github/workflows/build.yml

```diff
@@ -33,3 +33,56 @@ jobs:
               uses: matlab-actions/run-tests@v1
               with:
                  select-by-folder: opentelemetry-matlab/test
+    build-and-run-tests-windows:
+        runs-on: windows-latest
+        env:
+            OPENTELEMETRY_MATLAB_INSTALL: "${{ github.workspace }}/otel_matlab_install"
+        steps:
+            - name: Download OpenTelemetry-Matlab source
+              uses: actions/checkout@v3
+              with: 
+                path: opentelemetry-matlab
+            - name: Install MATLAB
+              uses: matlab-actions/setup-matlab@v1
+              with: 
+                release: R2023a
+            - name: Build OpenTelemetry-Matlab
+              run: |
+                  cd opentelemetry-matlab
+                  cmake --version
+                  cmake -S . -B build -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=${{ env.OPENTELEMETRY_MATLAB_INSTALL }}
+                  cmake --build build --config Release --target install -v
+            - name: Run tests
+              env:
+                 # Add the installation directory to the MATLAB Search Path by
+                 # setting the MATLABPATH environment variable.
+                 MATLABPATH: ${{ env.OPENTELEMETRY_MATLAB_INSTALL }}
+              uses: matlab-actions/run-tests@v1
+              with:
+                 select-by-folder: opentelemetry-matlab/test
+    build-and-run-tests-macos:
+        runs-on: macos-latest
+        env:
+            OPENTELEMETRY_MATLAB_INSTALL: "${{ github.workspace }}/otel_matlab_install"
+        steps:
+            - name: Download OpenTelemetry-Matlab source
+              uses: actions/checkout@v3
+              with: 
+                path: opentelemetry-matlab
+            - name: Install MATLAB
+              uses: matlab-actions/setup-matlab@v1
+              with: 
+                release: R2023a
+            - name: Build OpenTelemetry-Matlab
+              run: |
+                  cd opentelemetry-matlab
+                  cmake -S . -B build -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=${{ env.OPENTELEMETRY_MATLAB_INSTALL }}
```

Comment:
Add -DWITH_EXAMPLES=ON.

Reply:
-DWITH_EXAMPLES=ON has been added on Linux & Mac. Windows is currently experiencing failures Duncan is investigating.